### DISCUSSION TITLE
(Possible) minor mistakes in code examples

### DISCUSSION THREAD

**Discussion by simmotech** [dotnet/csharplang#6147] [General]
In
@https://github.com/dotnet/csharplang/blob/c5b687d7b56c8628738cb43f25aaf26c07549444/proposals/raw-string-literal.md
some of the code examples include a terminating semi-colon and some don't.

(search for "var json" to find the first one)

**Comment by CyrusNajmabadi** [comment]
The goal of the spec was to just demonstrate what is legal and what is not for a raw string. The surrounding syntax is not really a concern. 

That said, if you wanted to make a PR here to make these consistent, that would be fine.
